docs/content/index.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: home
 
-titleTemplate: Unstyled, accessible components for building high‑quality design systems and web apps in Vue
+titleTemplate: Unstyledd, accessible components for building high‑quality design systems and web apps in Vue
```

Comment:
@RayGuo-ergou  don't think it's working tho. I purposely have some typo here but the CI doesn't catch it or mention it.

Reply:
This due to this word has not yet in the typos dictionary. See: https://github.com/crate-ci/typos/blob/27ba9761d15d0f85acd8d0b500ac227e414d084d/CONTRIBUTING.md?plain=1#L32

e.g. it can detect 'donw'
![image](https://github.com/radix-vue/radix-vue/assets/33137074/d36372e6-e5df-41b7-9fce-e4a968db6505)


To have more precise typo check, [cspell ](https://github.com/streetsidesoftware/cspell) is a better choice but it's much slow and would be a bit painful to setup for large project. e.g. People's name repo's name etc.
![image](https://github.com/radix-vue/radix-vue/assets/33137074/e5bdf0ea-e34e-4ab6-9854-beed5df9573f)

Would you prefer cspell over typos?

Reply:
Actually I would prefer typos as it's fast, less false position and good enough for general typo check.

Reply:
Cool. Seems like it works. Yeah let's go with typo for now. 😁